ssz/cache/cache.py

```diff
@@ -0,0 +1,49 @@
+from collections.abc import (
+    MutableMapping,
+)
+from typing import (
+    TYPE_CHECKING,
+    Iterator,
+)
+
+from lru import LRU
+
+if TYPE_CHECKING:
+    MM = MutableMapping[bytes, bytes]
+else:
+    MM = MutableMapping
+
+
+class SSZCache(MM):
+    def __init__(self, cache_size: int=2048) -> None:
+        self._cache_size = cache_size
+        self.reset_cache()
+
+    def reset_cache(self) -> None:
```

Comment:
According to the [`collections.abc`](https://docs.python.org/3/library/collections.abc.html) classes, this should just be `clear` which is part of the core mutable mapping API.